src/EnergyPlus/DemandManager.hh

```diff
@@ -187,7 +192,9 @@ namespace DemandManager {
 			RotatedLoadNum( 0 ),
 			LowerLimit( 0.0 ),
 			UpperLimit( 0.0 ),
-			NumOfLoads( 0 )
+			NumOfLoads( 0 ),
+			FixedRate(0),
+			ReductionRatio(0)
 		{}
```

Comment:
@vidanovic (1) put a space between braces and the values (2) Use "0.0" instead of "0"